README.md

```diff
@@ -59,6 +59,21 @@ When I run the query above, I got the following result:
 
 ![Screenshot from Bigquery](data_period_screenshot.png)
 
+After running my first query above, I decided to run a query that would find out how many countries where recorded in my dataset 👇.
```

Comment:
Was it above or below. I am a tiny bit confused here. Make sure you use the right word. 😀

Reply:
Hi Katia, I have had a look and I have included two sets of code, that's why you may be confused as I refer back to the first code above when talking about the second code which is below. If it helps, I could remove the word 'above' from line 62?

Reply:
ok @Caitlin1306 there will be a merge conflict anyway. Try fixing it and let's see